admin-dev/themes/new-theme/js/pages/order/create/product-renderer.js

```diff
@@ -249,26 +243,16 @@ export default class ProductRenderer {
    *
    * @private
    */
-  _showResultBlock() {
+  showResultBlock() {
     $(createOrderMap.productResultBlock).removeClass('d-none');
   }
 
-  /**
```

Comment:
Why this block is removed (and not moved) ?

Reply:
Not used anywhere in the whole PrestaShop code :)